hardhat/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "3"
+services:
+  mintrally_localchain:
+    build:
+      context: .
+      dockerfile: ./Dockerfile
+    container_name: mintrally_localchain
+    ports:
+      - 8545:8545
+    volumes:
+      - ./:/usr/src/app
```

Comment:
Is ` command: npx hardhat node` not necessary?

Reply:
Yes you can
The way to run without Docker should write in a document thank you

Reply:
command: npx hardhat node are witten in Dockerfile.

Reply:
@terapasta 
What is the difference b/w write command on Dockerfile and docker-compose.yml? If there are critical issue it should be fixed.

Reply:
@yu23ki14 
Maybe you forgot to push something? I cannot find that command in Dockerfile.

I'm sorry if I've misunderstood you🙏

Reply:
Oh, that's right, some how it was gone. I fixed